src/modules/Modal/Modal.js

```diff
@@ -364,6 +372,7 @@ class Modal extends Component {
         onMount={this.handlePortalMount}
         onOpen={this.handleOpen}
         onUnmount={this.handlePortalUnmount}
+        style={{ display: 'flex !important' }}
```

Comment:
I see a problem there, React doesn't support `important` keyword for inline styles, see facebook/react#1881. Take a look on `Dimmer`'s PR #2656 for a workaround.

Reply:
Ah. I guess it is working in this case because this is being applied to a rootNode of a portal. Or possibly because the `toInlineStyle` package is handling it. I'll take a look today.

Reply:
@layershifter Updated. Technically it was working because we were passing those attributes as a string. This will keep the semantics the same way as we are handling this use case in `Dimmer`.